### DISCUSSION TITLE
kubectl port-forward permanently

### DISCUSSION THREAD

**Discussion by kevinzezel** [milvus-io/milvus#16764] [Q&A and General discussion]
Hi,

How can change milvuscluster_default.yaml to port-forward milvus cluster permanently? 

I'm looking for a way to not need to run this command all the time:

kubectl port-forward service/my-release-milvus 19530

Thanks,

**Comment by LoveEachDay** [comment]
You must rerun `kubectl port-forward` when your pod is restarted. 
You can change the milvus service type to `NodePort` or `LoadBalancer` and access milvus with the specified `<worker-node>:<port> `.

**Comment by Optimubin** [comment]
kubectl port-forward service/my-release-milvus 19530:19530
